content/applications/hr/lunch/user-accounts.rst

```diff
@@ -0,0 +1,106 @@
+to any monetary accounts or billing. Money **cannot** be transferred from users' bank accounts, nor
+users' credit cards be charged.
+
+Odoo's *Lunch* application **only** allows for manual entries of cash exchanges that are handled by
+the *Lunch* app manager. It is up to each individual company to create the method with which lunch
+accounts are replenished.
+
+.. example::
+   Some examples of how money can be organized and transferred within a company:
+
+   - Cash is handed to the *Lunch* app manager, who then updates the user's account.
+   - Money is automatically deducted from the user's paychecks, then the *Lunch* app manager updates
+     the account when paychecks are issued. This requires :ref:`adding a salary attachment
+     <payroll/worked-days-inputs>` for the user's payslip in the *Payroll* app.
+   - Companies can sell "lunch tickets" at a set price (for example, one ticket costs $5.00). Users
+     can purchase tickets from a *Lunch* app manager, who then updates the user's account.
+
+.. _lunch/cash-moves:
+
+Cash Moves
+==========
+
+To add funds to user accounts, each cash move must be individually logged. To view all cash move
+records, or create a new cash move, navigate to :menuselection:`Lunch app --> Manager --> Cash
+Moves`.
+
+All cash moves are presented in a list view, displaying each record's :guilabel:`Date`,
+:guilabel:`User`, :guilabel:`Description`, and :guilabel:`Amount` for each record. The total of all
+the cash moves is displayed at the bottom of the :guilabel:`Amount` column.
+
+.. image:: user-accounts/cash.png
+   :align: center
+   :alt: The list view of all cash moves.
+
+Add funds
+---------
+
+To add funds to a lunch account, click the :guilabel:`New` button, located in the top-left corner of
+the :guilabel:`Cash Moves` dashboard.
+
+A blank :guilabel:`Cash Moves` form loads. Enter the following information on the form:
+
+- :guilabel:`User`: select the user depositing cash into their account from the drop-down menu. If
+  the user is not in the database, they can be created by clicking either :guilabel:`Create "user"`
+  or :guilabel:`Create and edit...` to create the user and edit the :guilabel:`Create User` form.
+- :guilabel:`Date`: using the calendar popover, select the date the transaction occurred.
+- :guilabel:`Amount`: enter the amount being added to the lunch account.
+- :guilabel:`Description`: enter a brief description of the transaction.
+
+.. image:: user-accounts/cash-move-form.png
+   :align: center
+   :alt: The cash move form filled out for a transaction of $40.00.
+
+The cash move form automatically saves as data is entered, but the form can be saved manually at any
+time by clicking the *save manually* option, represented by a :guilabel:`(cloud upload)` icon,
+located in the top-left corner of the page. To delete the record and return to the *Cash Moves*
+dashboard, click the :guilabel:`↺ (Discard changes)` icon.
```

Comment:
I'm suggesting removing this portion, as this basic level of functionality is fairly rudimentary, and present on almost every page throughout the database. Also, I'm sure there's a 'Getting Started' doc that explains the functionality/purpose of those buttons. But I genuinely appreciate your attention to detail and desire to paint a complete picture for the reader -- however, in this case, I personally don't think it's super necessary.
```suggestion

```